src/tests/CommandHandlerTests.cpp

```diff
@@ -25,6 +25,40 @@ TEST(CommandHandler_Test, TestMissingBrokerSetToDefault) {
             "192.168.10.11:9092");
 }
 
+TEST(CommandHandler_Test, MissingTimeInHandleNewCommand) {
+  std::ifstream CommandFile(std::string(TEST_DATA_PATH) +
+                            "/msg-cmd-new-01.json");
+  nlohmann::json Command;
+  CommandFile >> Command;
+  EXPECT_EQ(findTime(Command, "start_time").count(), -1);
+  EXPECT_EQ(findTime(Command, "stop_time").count(), -1);
+}
+
+TEST(CommandHandler_Test, FindTimeHandleNewCommand) {
+  std::ifstream CommandFile(std::string(TEST_DATA_PATH) +
+                            "/msg-cmd-new-00.json");
+  nlohmann::json Command;
+  CommandFile >> Command;
+  EXPECT_EQ(findTime(Command, "start_time").count(), 123456789);
+  EXPECT_EQ(findTime(Command, "stop_time").count(), 123456790);
+}
+
+TEST(CommandHandler_Test, MissingTimeHandleStopCommand) {
+  std::ifstream CommandFile(std::string(TEST_DATA_PATH) +
+                            "/msg-conf-stop.json");
+  nlohmann::json Command;
+  CommandFile >> Command;
+  EXPECT_EQ(findTime(Command, "stop_time").count(), -1);
+}
+
+TEST(CommandHandler_Test, FindTimeHandleStopCommand) {
```

Comment:
As you are reading a relatively simple JSON structure here, I suggest that you make it a part of the C++ code instead of  reading it from file. You can do that by [using raw string literals.](https://en.cppreference.com/w/cpp/language/string_literal)

Reply:
:+1: